client/layout/guided-tours/positioning.js

```diff
@@ -34,11 +34,11 @@ const helpers = {
 	yBelow: bottom => {
 		return bottom + DIALOG_PADDING;
 	},
-	xAboveBelow: ( left, right, width ) => {
+	xAboveBelow: ( left, right ) => {
```

Comment:
Would probably make sense to remove that passed in `rect.width` where `xAboveBelow` is used.